src/router/index.js

```diff
@@ -0,0 +1,22 @@
+import VueRouter from "vue-router";
+
+import LogInPage from "@/views/LogInPage.vue";
+
+const routes = [
+  {
+    path: "/",
+    name: "LogIn",
+    component: LogInPage,
+  },
+  {
+    path: "*",
+    redirect: "/404",
```

Comment:
While this works, keep in mind we should also define a route for the `/404` path for it to work properly 😄